lib/pbench/server/api/resources/query_apis/metadata_index/__init__.py

```diff
@@ -138,7 +138,17 @@ def get_aggregatable_fields(
                 self.get_aggregatable_fields(v, f"{prefix}{f}.", result)
         return result
 
-    def get_mappings(self, document):
+    def get_mappings(self, document: JSON):
+        """
+        Utility function to return ES mappings by querying the Template
+        database against a given index.
+
+        :Args
```

Comment:
Your colon ended up on the wrong end of "Args".  :-)

Reply:
Ah, yeah